apps/site/assets/ts/ui/autocomplete/helpers.ts

```diff
@@ -0,0 +1,58 @@
+import { OnStateChangeProps } from "@algolia/autocomplete-js";
+import { ContentItem, Item, RouteItem, StopItem } from "./__autocomplete";
+import { isLGDown } from "../../helpers/media-breakpoints";
+
+function isStopItem(x: Item): x is StopItem {
+  return Object.keys(x).includes("stop");
+}
+
+export function isRouteItem(x: Item): x is RouteItem {
+  return Object.keys(x).includes("route");
+}
+
+export function isContentItem(x: Item): x is ContentItem {
+  return Object.keys(x).includes("_content_type");
+}
+
+export const getItemType = (item: Item): string => {
```

Comment:
This function doesn't seem to be used anywhere

Reply:
nice! will remove